src/components/map/Map.tsx

```diff
@@ -94,6 +99,11 @@ export const Map = ({
           </Marker>
         ))}
       </MapView>
+      {isMaximizeButtonVisible && (
+        <TouchableOpacity style={stylesForMap().maximizeMapButton} onPress={onMaximizeButtonPress}>
+          <Icon.NamedIcon name="expand" color={colors.primary} size={normalize(18)} />
```

Comment:
i will move this to configs/Icon.tsx, to have icons in one place. we need to adjust `name`s across different cities according to their selected icon font set, which should take place in one file at best.

Reply:
this will become `<Icon.ExpandMap size={normalize(18)} />` here